xml/System/String.xml

```diff
@@ -1616,7 +1618,7 @@ You can download the [Sorting Weight Tables](https://www.microsoft.com/download/
  The following example calls the <xref:System.String.Compare%28System.String%2CSystem.String%29> method to compare three sets of strings.  
```

Comment:
There are still quite a few examples for Compare that could be made interactive:
https://review.docs.microsoft.com/en-us/dotnet/api/system.string.compare?view=netframework-4.8&branch=pr-en-us-2971

Reply:
@mairaw -- Maira, should I include the 16 remarks.cs snippets as interactive?  They have no output, but they can be played with.  Here's the first such snippet:
```
// System.String.Compare(System.String,System.Int32,System.String,System.Int32,System.Int32)
public class CompareSample1_1
{
    //<Snippet2>
    static bool IsFileURI(String path)
    {
        return (String.Compare(path, 0, "file:", 0, 5, true) == 0);
    }
    //</Snippet2>
}
```

Reply:
No @WilliamAntonRohm, just the ones with output